gamemode/modules/base/sh_util.lua

```diff
@@ -97,23 +97,25 @@ function DarkRP.findPlayers(info)
     local InfoPlayers = {}
     for A in string.gmatch(info .. ";", "([a-zA-Z0-9:_.]*)[;(,%s)%c]") do
         if A ~= "" then
-            table.insert(InfoPlayers, A)
+            InfoPlayers[#InfoPlayers+1] = A
         end
     end
 
-    for _, PlayerInfo in ipairs(InfoPlayers) do
+    for _ = 1, #InfoPlayers do
+        local PlayerInfo = InfoPlayers[_]
         -- Playerinfo is always to be treated as UserID when it's a number
         -- otherwise people with numbers in their names could get confused with UserID's of other players
         if tonumber(PlayerInfo) then
             if IsValid(Player(PlayerInfo)) and not found[Player(PlayerInfo)] then
```

Comment:
While you're here, cache the return of Player(PlayerInfo) instead of calling it 4 times per iteration.

Reply:
This is a good idea :+1: